app/modules/intelligence/provider/provider_service.py

```
@@ -1,19 +1,23 @@
 import logging
 import os
+import asyncio
```

Ruff (0.8.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)